docs/userguide/userguide.md

```diff
@@ -3,6 +3,7 @@
 ```{toctree}
 Campaigns <campaigns>
 Constraints <constraints>
+Environment Vars <envvars>
```

Comment:
I'd prefer the full name "Environment variables"

Reply:
This was chosen not to grow the sidebar out of proportion

Reply:
Ok that makes sense